### PR TITLE
codespell: workflow, config, typos fixed

### PR DESCRIPTION
**Pull request by yarikoptic**
While listening to @elezar  giving a presentation, decided to check the CDI out: Having typos free code would be very useful for 1.0 release ;)

### CODE REVIEW

**Review by elezar — APPROVED** [review]
Thanks @yarikoptic. lgtm

@klihub @bart0sh any objections?

File: .codespellrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[codespell]
+skip = .git,*.pdf,*.svg
+# BU - part of JP3t17pCcGlemwknint6hfoeCVQrEMVwxRLRjXpq+BU=
```
Comment (elezar) [inline .codespellrc:3]: @yarikoptic as someone who is not too familiar with codespell what does this represent?

**Comment by yarikoptic** [comment]
merged `main`, resolved conflicts

File: .codespellrc (hunk 1 above)
Comment (elezar) [reply .codespellrc:3]: This string is in a `go.sum` file. Does it not make more sense to add `*.sum` to the `skip` list?

**Review by elezar — CHANGES_REQUESTED** [review]
Thanks for the explaination. Please rather add `go.sum` and `go.mod` to the skip list instead of handling the checksum as a word.

File: .codespellrc (hunk 1 above)
Comment (yarikoptic) [reply .codespellrc]: indeed, done!

**Review by elezar — APPROVED** [review]
lgtm

**Pull request merged**